share/completions/git.fish

```diff
@@ -645,5 +645,32 @@ complete -f -c git -n '__fish_git_using_command clean' -s x -d 'Remove ignored f
 complete -f -c git -n '__fish_git_using_command clean' -s X -d 'Remove only ignored files'
 # TODO -e option
 
+### git blame
+complete -f -c git -n '__fish_git_needs_command' -a blame -d 'Show what revision and author last modified each line of a file'
+complete -f -c git -n '__fish_git_using_command blame' -s b -d 'Show blank SHA-1 for boundary commits'
+complete -f -c git -n '__fish_git_using_command blame' -l root -d 'Do not treat root commits as boundaries'
+complete -f -c git -n '__fish_git_using_command blame' -l show-stats -d 'Include additional statistics at the end of blame output'
+complete -f -c git -n '__fish_git_using_command blame' -s L -d 'Annotate only the given line range'
+complete -f -c git -n '__fish_git_using_command blame' -s l -d 'Show long rev'
+complete -f -c git -n '__fish_git_using_command blame' -s t -d 'Show raw timestamp'
+complete -f -c git -n '__fish_git_using_command blame' -s S -d 'Use revisions from revs-file instead of calling git-rev-list'
+complete -f -c git -n '__fish_git_using_command blame' -l reverse -d 'Walk history forward instead of backward'
+complete -f -c git -n '__fish_git_using_command blame' -s p -l porcelain -d 'Show in a format designed for machine consumption'
+complete -f -c git -n '__fish_git_using_command blame' -l line-porcelain -d 'Show the porcelain format'
+complete -f -c git -n '__fish_git_using_command blame' -l incremental -d 'Show the result incrementally'
+complete -f -c git -n '__fish_git_using_command blame' -l encoding -d 'Specifies the encoding'
+complete -f -c git -n '__fish_git_using_command blame' -l contents -d 'Annotates the changes starting backwards from the working tree copy'
+complete -f -c git -n '__fish_git_using_command blame' -l date -d 'Specifies the format used to output dates'
+complete -f -c git -n '__fish_git_using_command blame' -s M -d 'Detect moved or copied lines within a file'
+complete -f -c git -n '__fish_git_using_command blame' -s C -d 'Detect lines moved or copied from other files that were modified in the same commit'
+complete -f -c git -n '__fish_git_using_command blame' -s h -d 'Show help message'
+complete -f -c git -n '__fish_git_using_command blame' -s c -d 'Use the same output mode as git-annotate'
+complete -f -c git -n '__fish_git_using_command blame' -s f -l show-name -d 'Show the filename in the original commit'
+complete -f -c git -n '__fish_git_using_command blame' -s n -l show-number -d 'Show the line number in the original commit'
+complete -f -c git -n '__fish_git_using_command blame' -s s  -d 'Suppress the author name and timestamp from the output'
+complete -f -c git -n '__fish_git_using_command blame' -s e -l show-email  -d 'Show the author email instead of author name'
+complete -f -c git -n '__fish_git_using_command blame' -s w -d 'Ignore whitespace when comparing the parent’s version and the child’s to find where the lines came from'
```

Comment:
Just "Ignore whitespace changes" should be enough. Keeping descriptions short allows us to use more columns. In general you probably want to paraphrase what the git man pages say as they have a rather... special style.